apps/devhub/views.py

```diff
@@ -874,10 +883,20 @@ def upload_validation_context(request, upload, addon_slug=None, addon=None,
 
 @login_required
 def upload_detail(request, uuid, format='html'):
-    upload = get_object_or_404(FileUpload, uuid=uuid)
-
     if format == 'json' or request.is_ajax():
-        return json_upload_detail(request, upload)
+        with statsd.timer('devhub.upload_detail'):
```

Comment:
This one is at https://app.datadoghq.com/metric/explorer?live=true&page=0&is_auto=false&from_ts=1452542782736&to_ts=1452629182736&tile_size=m&exp_metric=addons_prod.view.devhub.views.upload_detail.GET.avg&exp_scope=&exp_agg=avg&exp_row_type=metric